api/api.gen.go

```diff
@@ -126,7 +132,7 @@ const (
 
 // Defines values for NotificationChannelWebhookCreateRequestType.
 const (
-	WEBHOOK NotificationChannelWebhookCreateRequestType = "WEBHOOK"
+	NotificationChannelWebhookCreateRequestTypeWEBHOOK NotificationChannelWebhookCreateRequestType = "WEBHOOK"
```

Comment:
This is only used for deserializing/serializing by the API itself, so it should be fine to leave it like this.